.rspec

```diff
@@ -1 +1,2 @@
---color
\ No newline at end of file
+--color
+--require 'pry'
```

Comment:
We got binding.irb now though right? Do we even need pry at all?

Reply:
No because that broke jruby which is officially still supported.
https://github.com/railslove/cmxl/pull/61